esbuild.config.js

```diff
@@ -5,6 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 const { build } = require('esbuild');
+const esbuildPluginPino = require('esbuild-plugin-pino');
```

Comment:
I think this was one of the primary issues.  In order to have anything using pino you need to include this pino plugin ( I think, this is more @mingxuanzhangsfdx area of expertise).